src/app/legacy/containers/Gist/index.jsx

```diff
@@ -88,7 +84,7 @@ const componentsToRender = (service, script, dir) => ({
             script={script}
             direction={dir === 'rtl' ? 'right' : 'left'}
             bulletPointShape="square"
-            bulletPointColour={C_GREY_6}
+            bulletPointColour={GREY_6}
```

Comment:
Used a direct import as I wasn't sure if I could/ how to use useTheme() hook.

Reply:
In the gist component you could set the `bulletPointColour` and pass it through as an argument:

```jsx
const Gist = ({ blocks }) => {
  const { service, script, dir, translations } = useContext(ServiceContext);
  const {
    palette: { GREY_6: bulletPointColour },
  } = useTheme();
  const gistTitle = pathOr('At a glance', ['gist'], translations);
  return (
    <GridItemLarge role="region" aria-labelledby="gist-title">
      <GistWrapper service={service} script={script} dir={dir}>
        <GistIntroduction service={service} script={script} id="gist-title">
          {gistTitle}
        </GistIntroduction>
        <Blocks
          blocks={blocks}
          componentsToRender={componentsToRender(service, script, dir, bulletPointColour)}
        />
      </GistWrapper>
    </GridItemLarge>
  );
};
```
And then in the component to render function
```jsx
const componentsToRender = (service, script, dir, bulletPointColour) => ({
  text: props => (
    <Text
      {...props}
      componentsToRender={{
        unorderedList: innerProps => (
          <GistList
            {...innerProps}
            service={service}
            script={script}
            direction={dir === 'rtl' ? 'right' : 'left'}
            bulletPointShape="square"
            bulletPointColour={bulletPointColour}
          />
```